src/main/java/org/usfirst/frc4904/robot/commands/ElevatorDown.java

```diff
@@ -0,0 +1,20 @@
+
+package org.usfirst.frc4904.robot.commands;
+
+import org.usfirst.frc4904.robot.RobotMap;
+import org.usfirst.frc4904.standard.commands.motor.MotorPositionConstant;
+import org.usfirst.frc4904.robot.subsystems.Elevator;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+/**
+ puts elevator at down position
+ */
+public class ElevatorDown extends MotorPositionConstant {
```

Comment:
This should extend ElevatorSet, not MotorPositionConstant.

Reply:
this means that you will also need to change your supers()